qiskit_ibm_runtime/api/clients/runtime.py

```diff
@@ -274,27 +271,17 @@ def session_details(self, session_id: str) -> Dict[str, Any]:
         """
         return self._api.runtime_session(session_id=session_id).details()
 
-    def list_backends(
-        self, hgp: Optional[str] = None, channel_strategy: Optional[str] = None
-    ) -> List[str]:
+    def list_backends(self, hgp: Optional[str] = None) -> List[str]:
```

Comment:
The docstring sounds like we will return the backend instances, not the names (List[str]).